files/tasks/packit_reverse_dep_test.yaml

```diff
@@ -17,7 +17,11 @@
   command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-install-requirements-pip.yaml
   args:
     chdir: "{{ reverse_dir }}"
-- name: install and run packit tests via ansible playbooks
+- name: install and run basic packit tests via ansible playbooks
   command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-tests.yaml
   args:
     chdir: "{{ reverse_dir }}"
+- name: install and run packit tests with session recording via ansible playbooks
+  command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-tests-session-recording.yaml
```

Comment:
Thats good idea, but I have trouble with this runnning ansible as part of anoter ansible here, for setup steps it is fine, but look at output in logs: https://softwarefactory-project.io/logs/64/64/65ef28d8c86f7cc61ad1b57e7dd9884a49174533/check/requre-reverse-dep-packit-tests/a75612c/job-output.txt.gz
ansible removed EOLs and resutls are very unreadable. thats why I had there directly called ``python -v tests`` 
Is there any ansible way to provide better output of testing using ansible inside ansible?

Reply:
I know about that.

- I only corrected the target names and remove the redundant pytest task. (Shall we improve later?)
- We discussed that with Tomat and:
  - use some debug mode to have better output
  - zuul is designed to do the tests on multiple repos, so we can ask them if there is some easy way for running dep-tests.

Reply:
okay, then mergeit! ;-)

Reply:
Thanks -- can you create an issue for that?

Reply:
Otherwise I do that also in this way, to ensure, that packit  ansible yamls will not reinstall requre as part of testing.